go.mod

```diff
@@ -4,12 +4,12 @@ go 1.16
 
 require (
 	github.com/alecthomas/kong v0.2.17
-	github.com/datastax/astra-client-go/v2 v2.2.9 // indirect
-	github.com/datastax/go-cassandra-native-protocol v0.0.0-20211124104234-f6aea54fa801
+	github.com/datastax/astra-client-go/v2 v2.2.54
+	github.com/datastax/go-cassandra-native-protocol v0.0.0-20220706104457-5e8aad05cf90
 	github.com/hashicorp/golang-lru v0.5.4
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.1
 	go.uber.org/atomic v1.8.0
 	go.uber.org/multierr v1.7.0 // indirect
```

Comment:
hmm it looks like `go fmt` should be run or `go mod tiny`

Reply:
`go fmt` and `go mod tidy` do not complain. I have run them before submitting PR.

Reply:
After upgrading go to 1.18, indeed `tidy` was needed.